releasenotes/notes/add-basistranslator-to-default-levels-1c0de250f8ca4a59.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
```

Comment:
I think we should specifically highlight the ability to translate to other basis now (and specifically CZ and iSWAP). It should be in the prelude of the releasenotes actually.

Reply:
:+1: added a note to the prelude, and CX->CZ and CX->iSwap bell tests.